src/main/java/com/vinyl/service/validation/EmailAndPasswordValidator.java

```diff
@@ -13,6 +13,8 @@
 
 	private final UserRepository userRepository;
 	private final BCryptPasswordEncoder passwordEncoder;
+	
+	private User user;
```

Comment:
you dont seem to be using this field, remove it

Reply:
done